airflow/contrib/hooks/gcs_hook.py

```diff
@@ -543,11 +549,12 @@ def create_bucket(self,
             raise ValueError('Bucket names must end with a number or letter.')
 
         service = self.get_conn()
-        bucket_resource = {
+        bucket_resource = copy.copy(resource)
+        bucket_resource.update({
```

Comment:
If _resource_ is None, _bucket_resource_ is None. It can't update a NoneType object.